sandbox/rocky/tf/plotter/plotter.py

```diff
@@ -0,0 +1,108 @@
+import atexit
+from collections import namedtuple
+import enum
+from enum import Enum
+from multiprocessing import Process
+import platform
+from queue import Empty, Queue
+from threading import Thread
+
+import numpy as np
+import tensorflow as tf
+
+from rllab.sampler.utils import rollout
+
+__all__ = ["Plotter"]
```

Comment:
This is not required if  * is not used in the import. See [here](https://docs.python.org/3.5/tutorial/modules.html#importing-from-a-package) for further information.